scormxblock/scormxblock.py

```diff
@@ -328,7 +337,7 @@ def scorm_get_values(self, request=None, suffix=None):
                 data[key] = value
 
         values.update(data)
-        return Response(json.dumps(values), content_type='application/json')
+        return values
```

Comment:
Will json_handler add the proper content_type and serialization? @luismorenolopera

Reply:
json_handler takes care of serializing with the appropriate content_type in case they are not provided https://github.com/edx/XBlock/blob/master/xblock/mixins.py#L36

Reply:
Ok, thanks. @luismorenolopera